lvsfunc/misc.py

```diff
@@ -186,6 +186,7 @@ def limit_dark(clip: vs.VideoNode, filtered: vs.VideoNode,
     :return:                  Conditionally filtered clip
     """
     def _diff(n, f, clip, filtered, threshold, threshold_range):
+        # TODO: Use unused param `n`.
```

Comment:
I think `n` is required to be passed to frame eval, even if it ends up being unused.

Reply:
It is and this isn't even a linter warning, so yeah this comment should be removed